src/main/scala/org/broadinstitute/hail/driver/ExportPlink.scala

```diff
@@ -25,6 +58,85 @@ object ExportPlink extends Command {
 
   def requiresVDS = true
 
+  def getFamRows(vds: VariantDataset, options: Options) : IndexedSeq[String] = {
```

Comment:
OK, so I refactored the code going this direction. Since I feel like a user should be able to super-easily get back the data he imported, I still kept an alternative implementation of the sex and phenotype corresponding to the defaults when using annotatesamples fam. Now if the user specifies anything that's not default, then I totally agree with you and changed the code so that it's clear what's expected.